backend/src/main/java/atlantbh/restaurants/services/LocationService.java

```diff
@@ -0,0 +1,24 @@
+package atlantbh.restaurants.services;
+
+import atlantbh.restaurants.models.Location;
+import atlantbh.restaurants.repositories.LocationRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class LocationService extends BaseService<Location, LocationRepository> {
+
+    public Location findLocationByCountryAndCity(String country, String city) {
+        return repository.findByCountryAndCity(country,city);
+    }
+    public List<Location> findAllCitiesForCountry(String country) {
+        List<Location> list = repository.findAllCitiesByCountry(country);
+        return list;
```

Comment:
since you don't change this variable, just return repository.find...
and same for method below.

Reply:
Fixed c2ed9b4060983e1312e68ee9cdd1de2be1986207